pandas/core/reshape/concat.py

```diff
@@ -95,6 +97,7 @@ def concat(
     verify_integrity: bool = False,
     sort: bool = False,
     copy: bool = True,
+    initializer: NDFrame | None = None,
```

Comment:
In my opinion the new parameters are better be introduced using keyword-only argument notation (for better maintainability in future).

```
def func(first, second, *, keyword_only=None):
```

I understand that there are already multiple kwargs, which are not keyword-only and can technically be passed as positional, but maybe it is a good moment to start (question to core developers)?

Reply:
@ivanovmg, could you provide an example of how the function signature would look like with keyword-only argument notation?

Admittedly I'm still relatively new to writing production code so it would really help if you could show me an example of how it would be done. Thanks!

Reply:
Like this:
```
def concat(
    objs: Iterable[NDFrame] | Mapping[Hashable, NDFrame],
    axis=0,
    join="outer",
    ignore_index: bool = False,
    keys=None,
    levels=None,
    names=None,
    verify_integrity: bool = False,
    sort: bool = False,
    copy: bool = True,
    *,  # <- note star here
    initializer: NDFrame | None = None,
) -> FrameOrSeriesUnion:
```

But, again, I have not seen this approach widely used among the code base, so I am not sure what are the core development team preferences.

Reply:
nevertheless, learnt something new today. Thank you :)